graylog2-web-interface/src/components/inputs/InputListItem.jsx

```diff
@@ -139,22 +140,21 @@ const InputListItem = React.createClass({
     }
 
     const inputForm = definition ?
-        <InputForm ref="configurationForm" key={'edit-form-input-' + input.id}
+        <InputForm ref="configurationForm" key={`edit-form-input-${input.id}`}
                    globalValue={input.global} nodeValue={input.node}
                    configFields={definition.requested_configuration}
-                   title={'Editing Input ' + input.title}
+                   title={`Editing Input ${input.title}`}
                    titleValue={input.title}
                    typeName={input.type} includeTitleField
                    submitAction={this._updateInput} values={input.attributes} /> : null;
 
     const additionalContent = (
       <div>
         <Col md={8}>
-          <Well bsSize="small" className="configuration-well">
-            <ul>
-              {this._getConfigurationOptions(this.props.input.attributes)}
```

Comment:
We should also remove the `_getConfigurationOptions` method, as it is not used any more.